### DISCUSSION TITLE
Awesome Project and interested to contribute

### DISCUSSION THREAD

**Discussion by piyusharorawork** [webstudio-is/webstudio-community#4] [Ideas]
I love this idea and I am definitely interested in contributing to this project . 
Also some ideas which might interest you: 

1. Figma plugin to convert design to web studio 
2. Create desktop electron app for web studio 
3. Support Other frame works than react may be in far future

Also please create a discord channel , it will be super helpful.

**Comment by kof** [comment]
Hey, absolutely, all 3 are on my mind! I am basically building it all so that: figma can be imported because we have an API and data format we can convert to, architecture is super portable, so that it can be packed as an offline-first desktop app, rendering on the canvas is designed so that we can provide primitives for any other framework on the future. 

There is a lot of work to get there, we need a team, it will take time, because we need to establish data standard and build offline syncing infra and multiplayer logic to make it all possible, but its top priority.

**Comment by ninjanordbo** [comment]
This is such a cool tool! Im very happy I found this mentioned on twitter, brilliant for collaboration! Definitely smart to get a discord set up.

**Comment by kof** [comment]
Cool, adding a discord!

**Reply by kof** [reply]
[Here we go](https://discord.gg/UNdyrDkq5r)
